cordformation/src/test/kotlin/net/corda/plugins/BaseformTest.kt

```diff
@@ -43,6 +43,8 @@ open class BaseformTest {
         installResource("repositories.gradle")
         installResource("gradle.properties")
         installResource("postgres.gradle")
+        installResource("external-service.gradle")
+        installResource("external-service-2.gradle")
```

Comment:
Are you replicating the pattern we used for `postgres.gradle`? The external database configuration has _a lot_ of complicated elements that is still largely "boilerplate", and so we separated out all of the "database dependent" parts into a script plugin. But I don't think we need this level of complexity for your `external` block.

Reply:
Initial story requirement called for a separate file. While it's true the external block will be, on average, rather short, there's some value in keeping it separated, in my opinion.

Reply:
> Initial story requirement called for a separate file. While it's true the external block will be, on average, rather short, there's some value in keeping it separated, in my opinion.

I'm not convinced that this separate file is going to be compatible with using Gradle's lazy APIs. In which case the separate file is going to have to lose, I'm afraid.

Reply:
From what I'm testing right now, it seems indeed incompatible with Gradle lazy init.